tests/unit/load-webpack.ts

```diff
@@ -0,0 +1,145 @@
+import * as assert from 'intern/chai!assert';
+import * as registerSuite from 'intern!object';
+import global from '../../src/global';
+import load from '../../src/load-webpack';
+
+interface WebpackModules {
+	[id: number]: any;
+}
+
+let webpackModules: WebpackModules;
+function setModules(modules?: { [mid: string]: any }) {
+	let nextId = 0;
+	webpackModules = Object.create(null) as WebpackModules;
+
+	if (modules) {
+		const idMap = Object.create(null);
+		Object.keys(modules).forEach((mid: string) => {
+			const id = nextId++;
+			idMap[mid] = { id, lazy: mid.indexOf('bundle!') > -1 };
```

Comment:
for the contents of the map, do you think we might want to add some mechanism for different behaviours per plugin perhaps? rather than just the lazy flag at the moment that we have for `bundle`?